Intersect (Core)/Config/PlayerOptions.cs

```diff
@@ -59,6 +59,15 @@ public class PlayerOptions
         /// </summary>
         public bool ShowLevelByName = false;
 
+        /// <summary>
+        /// First thanks for letting me share an idea, of losing XP when dying <3
+        /// </summary>
+        public int ExpLossOnDeathPercent = 0;
+        /// <summary>
```

Comment:
Please add a a line break before this summary so it matches the rest of the file aesthetically.

Reply:
Done